site/content/en/docs/handbook/addons/headlamp.md

```diff
@@ -24,15 +24,14 @@ minikube service headlamp -n headlamp
 To authenticate in Headlamp, fetch the Authentication Token using the following command:
 
 ```shell script
-export SECRET=$(kubectl get secrets --namespace headlamp -o custom-columns=":metadata.name" | grep "headlamp-token")
-kubectl get secret $SECRET --namespace headlamp --template=\{\{.data.token\}\} | base64 --decode
-``` 
+kubectl create token headlamp --duration 24h -n headlamp
```

Comment:
I don't think it should be 24h. For a development system this is a bit tedious to have to create a new token every day.

```
kubectl create token headlamp-admin -n kube-system
```

If leaving it without an expiry is too much, what about a line telling the users about removing the duration if their minikube is just for development?